docs/concepts/services-networking/service.md

```diff
@@ -427,6 +427,21 @@ with the user-specified `loadBalancerIP`. If the `loadBalancerIP` field is not s
 an ephemeral IP will be assigned to the loadBalancer. If the `loadBalancerIP` is specified, but the
 cloud provider does not support the feature, the field will be ignored.
 
+#### Internal load balancer
+In a mixed environment it is sometimes necessary to route traffic from services inside the same VPC (or equivalent on non AWS platforms).
+This can be achieved by adding the following annotation to the service:
```

Comment:
This is all AWS content. Even the annotation is AWS specific. IMO it makes more sense to  change section title to `Internal load balancer on AWS` that way it matches the AWS specific `SSL support on AWS` below.